administrator/language/de-DE/com_templates.ini

```diff
@@ -129,6 +129,7 @@ COM_TEMPLATES_FOLDER_NAME="Verzeichnisname"
 COM_TEMPLATES_FOLDER_NOT_EXISTS="Das Verzeichnis existiert nicht."
 COM_TEMPLATES_FTP_DESC="Für ein Update der Template-Quelldateien braucht Joomla! FTP-Zugangsdaten. Bitte diese in die unteren Formularfelder eingeben."
 COM_TEMPLATES_FTP_TITLE="FTP Anmelde Details"
+COM_TEMPLATES_GD_EXTENSION_NOT_AVAILALE="Die GD-Erweiterung für PHP ist nicht verfügbar.<br>Um Bilder zu manipulieren, müssen zusätzliche Erweiterungen auf dem Server installiert sein. Für Details, die <a href='https://downloads.joomla.org/technical-requirements'>Offiziellen technischen Anforderungen</a> in der Dokumentation lesen."
```

Comment:
Das würde ich gerne vereinheitlichen in den Sprachdateien.

Es gibt derzeit auch hier wieder mehrere Schreibweisen.

- Weitere Informationen können in der ... nachgelesen werden.
- Weitere Informationen sind in der ... zu finden.
- Weitere Informationen dazu können in der ... gefunden werden.

Was wollen wir @joomlagerman/reviewer und dann einheitlich?

Reply:
>     * Weitere Informationen sind in der ... zu finden.

:+1: